discord_bots/models.py

```diff
@@ -101,6 +105,18 @@ class Category:
         default=0,
         metadata={"sa": Column(Integer, nullable=False, server_default=text("0"))},
     )
+    sigma_decay_amount: float = field(
+        default=0.0,
+        metadata={"sa": Column(Float, nullable=False, server_default=text("0.0"))},
+    )
+    sigma_decay_grace_days: int = field(
```

Comment:
I'm still not sure if a grace period is useful or just added complexity. Is there a scenario you can think of where you'd want this set to anything other than 0?

Reply:
Not certain, but in honesty my initial thought was to set it to 1, which would mean that you never get decayed if you play every day. It doesn't make a whole lot of difference from a TrueSkill perspective, but from a psychological one it may be helpful?

Reply:
If you configure the job to run at 11:59 pm every night, then you wouldn't get decayed if you play that day. I do think it's nice to have the option as an extra dial to tune, but I'm still not entirely convinced it's necessary. 
A scenario where it would be nice, is if you want a very high sigma decay amount but a long grace period: say after 7 days of inactivity you begin decaying players heavily. I'm leaning more towards removing it, since I don't see an immediate usecase but my opinion isn't very strong

Reply:
On second thought, I think we should keep it. It's a useful parameter